kernel/rtlil.cc

```diff
@@ -344,9 +429,11 @@ std::string RTLIL::Const::decode_string() const
 
 bool RTLIL::Const::is_fully_zero() const
 {
+	bitvectorize();
+	bitvectype& bv = assert_get_bits("Const::decode_string");
```

Comment:
Why not just call `bits()` instead of repeating `bitvectorize(); bitvectype& bv = assert_get_bits(...)` everywhere?

Reply:
Ah you have a TODO to get rid of this